.gitmodules

```diff
@@ -1,4 +1,9 @@
- 
-[submodule ".test/ngs-test-data"]
-	path = .test/ngs-test-data
-	url = https://github.com/snakemake-workflows/ngs-test-data
+
+[submodule ".test/data/atacseq/test-datasets"]
```

Comment:
Maybe also add a comment here, that these are intended for testing with larger datasets once we have a dedicated GitHub Actions machine. Comments should be fine to include, using `#`. See [the docs for `.gitmodules`](https://www.git-scm.com/docs/gitmodules) that point to the [syntax of `git-config`](https://www.git-scm.com/docs/git-config#_syntax).